app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt

```diff
@@ -55,8 +56,17 @@ class LaunchActivity : AppCompatActivity() {
             withContext(Dispatchers.Main) {
                 when {
                     user == null -> loginUser()
-                    navigationArgs?.shouldLock != false && isKeyguardSecure() && localSettings.isAppLocked -> startAppLockActivity()
-                    else -> startApp()
+                    navigationArgs?.shouldLock != false && isKeyguardSecure() && localSettings.isAppLocked -> {
+                        startAppLockActivity()
```

Comment:
Won't this bug if `getMailboxesCount() == 0L` ?
Same in kDrive ?

Reply:
done 
(kDrive : https://github.com/Infomaniak/android-kDrive/pull/1006)